phoenix-core/src/it/java/org/apache/phoenix/end2end/BackwardCompatibilityIT.java

```diff
@@ -437,8 +472,14 @@ public void run() {
     }
 
     // Executes the SQL commands listed in the given operation file from the sql_files directory
-    private void executeQueriesWithCurrentVersion(String operation) throws Exception {
+    private void executeQueriesWithCurrentVersion(String operation,
+            boolean setMaxLookBackAge) throws Exception {
```

Comment:
@virajjasani Was wondering whether you would entertain the idea of using an EnumSet instead of boolean (which is specific to maxLookBaackAge). The thinking was we could then reuse this method more easily for other property checks and combinations of them.